modules/apps/data-engine/test.properties

```diff
@@ -10,4 +10,15 @@
 ##
 
     modules.includes.required.functional[relevant]=\
-        apps/app-builder
\ No newline at end of file
+        apps/app-builder,\
+        apps/document-library,\
+        apps/dynamic-data-lists,\
+        dynamic-data-mapping,\
+        apps/journal
```

Comment:
@alexcandidos pode remover o journal e document-library por enquanto, quando a flag for retirada depois da migração, botamos de volta. pode botar pra frente sem essas duas linhas.

Reply:
ok!